at-tanvo/Exercise2.rb

```diff
@@ -0,0 +1,13 @@
+def sum a
+  kiem_tra = a.all? {|i| i.is_a? Integer}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><table>
  <tbody>
    <tr>
      <td><span data-href="https://github.com/atruby-team/internship-18-summer/blob/47f87600d63703e4dcdf4c1477df3685adc982eb/at-tanvo/Exercise2.rb#L2"></span>Layout/SpaceInsideBlockBraces: Space between { and</td>
      <td>missing.</td>
    </tr>
  </tbody>
</table>
</td>
    </tr>
  </tbody>
</table>